manifest.json

```diff
@@ -10,6 +10,7 @@
   },
   "permissions": [
     "history",
```

Comment:
No, you need it if you want to suck in prior visits, which I heard was coming down the pike.